apm-agent-plugins/apm-asynchttpclient-plugin/src/test/java/co/elastic/apm/agent/asynchttpclient/AsyncHttpClientInstrumentationTest.java

```diff
@@ -30,8 +30,8 @@
 import java.io.IOException;
 import java.util.Arrays;
 
+import static co.elastic.apm.agent.testutils.assertions.Assertions.assertThat;
 import static org.asynchttpclient.Dsl.asyncHttpClient;
-import static org.assertj.core.api.Assertions.assertThat;
```

Comment:
[for reviewer] this is just test code refactoring allowed by the addition of `AbstractSpanAssert#isExit`.